.github/pull_request_template.md

```diff
@@ -1,23 +1,30 @@
+# Pull Requests
+
+A Pull Request should close a **single** issue; multiple issues can be closed when the issues are **small and related**, but that should be an exception not the rule. Please
+keep Pull Requests small and targeted; large 'code drops' with dozens of files will be closed and asked to split into reviewable pieces. Reviews that need to be large due to dependencies will be
+reviewed on a case-by-case basis.
+
 ## Related Issue
```

Comment:
noticed that header levels for sub-topics are different from the rest of the templates whereas all subtopics receive header level 1, but the PR template gets header level 1 and 2. Just wanted to make sure this is intended.

Reply:
Good catch!